cli/command/run/cmd.go

```diff
@@ -341,6 +387,7 @@ func runPlaywrightInDocker(p playwright.Project, testco testcomposer.Client) (in
 
 func runPlaywrightInSauce(p playwright.Project, regio region.Region, tc testcomposer.Client, rs resto.Client, as *appstore.AppStore) (int, error) {
 	log.Info().Msg("Running Playwright in Sauce Labs")
+	printTestEnv("sauce")
```

Comment:
@tianfeng92 was this here for debugging?

Reply:
Not for debugging. It's for print out the ASCII pic to show the env.